### PR TITLE
Move blog toc generation code to repo

### PR DESCRIPTION
**Pull request by kevinye-openstore**
- This code generates TOC from blog headings (h2 and h3) and create anchors to navigate to
- This code currently lives on Webflow, but has since grown to be complicated enough that requires version control

### CODE REVIEW

File: src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    const h2s = $('h2')
+
+    if (h2s.length > 0) {
+      const tocList = $('<ul></ul>')
+      tocList.addClass('toc-list')
+
+      h2s.each(function () {
+        const h2 = $(this)
+        h2.html
```
Comment (wenboyu2) [inline src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts:39]: what does this line do?

**Review by wenboyu2 — APPROVED** [review]

File: src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts (hunk 1 above)
Comment (gabrielyungun) [inline src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts:39]: remove?

**Review by gabrielyungun — APPROVED** [review]

File: src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts (hunk 1 above)
Comment (kevinye-openstore) [inline src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts:39]:
```suggestion
```

Comment (kevinye-openstore) [reply src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts]: Good catch. Was a part of debug

Comment (kevinye-openstore) [reply src/packages/scripts/generateTOCFromHeadings/generateTOCFromHeadings.ts]: Good catch. Was a part of debug

**Pull request merged**